### DISCUSSION TITLE
How use the library math.h in C?

### DISCUSSION THREAD

**Discussion by Isaias00C** [microsoft/vscode-discussions#1145] [Extension Development QnA]
to compiler a program in C using the library math.h need to use 'gcc <name> -o <name> -lm'. But I can't use this '-lm'

![Captura de tela de 2024-04-17 09-25-51](https://github.com/microsoft/vscode-discussions/assets/167316633/11a58f04-65bd-4b76-853d-f2d5810280ac)

in line 6 I write the '-lm' using the extension code runner, but it not solved my problem.

**Comment by theReynald** [comment]
Hi @Isaias00C! 

Can you try this in VS Code Insiders to see if you're having the same issue? 

Also, in addition to the screen shot, can you paste the settings.json "code-runner .executorMap" code in your post along with the text result from the terminal? This makes it easier for us to copy what you have and research it.

**Comment by burkeholland** [comment]
This question appears to be off topic for VS Code extension development. Stack Overflow would be a better place for this. Closing for now. If you feel this is a mistake, please feel free to reopen.
